# Why?
# ==> In setting `target-os` is set to 'linux_android' but here it checks for `linux-android`.

--- ghc/hadrian/src/Oracles/Setting.hs	2023-10-07 08:28:11.000000000 +0530
+++ ghc.mod/hadrian/src/Oracles/Setting.hs	2024-01-31 21:08:09.510592353 +0530
@@ -280,7 +280,7 @@
 elfOSes :: [String]
 elfOSes =
     [ "linux", "freebsd", "dragonfly", "openbsd", "netbsd", "solaris2", "kfreebsdgnu"
-    , "haiku", "linux-android"
+    , "haiku", "linux_android"
     ]
 
 -- | List of OSes that use the Mach-O object format.
